docs/runtime-contract.md

```diff
@@ -82,9 +82,9 @@ serverless platform SHOULD observe the following properties:
 
 ### State
 
-The general OCI state may not be available for introspection within the
-container, and may only be visible to the system operator or platform provider.
-In a highly-shared environment, containers may experience the following:
+The general OCI state might not be available for introspection within the
+container, and might only be visible to the system operator or platform provider.
```

Comment:
I think `MAY` might be correct here, but I think @dgerd has thought about this more recently than I.

Reply:
I would actually prefer removing this as it:

1. Seems duplicative with https://github.com/knative/serving/blob/master/docs/runtime-contract.md#operations
2. Provides little guidance/value to users or platform providers

Reply:
Since I thought it was non-normative anyway, I'm ok with removing it - done!